stanford_media.info.yml

```
@@ -12,6 +12,7 @@ dependencies:
   - drupal:field
   - drupal:media
   - drupal:media_library
+  - drupal:oembed_providers
```

```suggestion
  - oembed_providers:oembed_providers
```
Also let's alphabetize the dependencies after this change.